invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/communities.js

```diff
@@ -0,0 +1,257 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2020-2021 CERN.
+// Copyright (C) 2020-2021 Northwestern University.
+// Copyright (C) 2021 Graz University of Technology.
+// Copyright (C) 2021 New York University.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import {
+  SearchAppFacets,
+  SearchAppResultsPane,
+} from "@js/invenio_search_ui/components";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import React, { useState } from "react";
+import {
+  Input,
+  Card,
+  Container,
+  Grid,
+  Label,
+  Item,
+  Button,
+  Segment,
+} from "semantic-ui-react";
+import _truncate from "lodash/truncate";
+import {
+  BucketAggregation,
+  SearchBar,
+  Count,
+  Sort,
+  ResultsList,
+  Pagination,
+  ResultsPerPage,
+} from "react-searchkit";
+import {
+  RDMBucketAggregationElement,
+  RDMRecordFacetsValues,
+  SearchHelpLinks,
+} from "../../search/components";
+
+function ResultsGridItemTemplate({ result, index }) {
+  return (
+    <Card fluid key={index} href={`/communities/${result.metadata.id}`}>
+      <Card.Content>
+        <Card.Header>{result.metadata.title}</Card.Header>
+        <Card.Description>
+          <div
+            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
+          />
+        </Card.Description>
+      </Card.Content>
+    </Card>
+  );
+}
+
+export function CommunitiesResultsItemTemplate({ result, index }) {
+  return (
+    <Item key={index}>
+      {index == 0 ? (
+        <Item.Image size="tiny" src={result.links.logo} />
+      ) : (
+        <Item.Image size="tiny" src="/static/images/placeholder.png" />
+      )}
+      <Item.Content>
+        <Item.Header href={`/communities/${result.id}`}>
+          {result.metadata.title}
+          {result.metadata.type && (
+            <span style={{ marginLeft: "5px" }}>
+              <Label size="small">{result.metadata.type}</Label>
+            </span>
+          )}
+        </Item.Header>
+        {result.metadata.website && (
+          <Item.Description href={result.metadata.website} target="_blank">
+            <a
+              dangerouslySetInnerHTML={{
+                __html: result.metadata.website,
+              }}
+            ></a>
```

Comment:
hyperlink should be a part of a tag probably. I am not sure what will be the outcome on dangerouslySetInnerHTML set on `<a>` tag, is it needed? will it miss the url description?
```suggestion
          <Item.Description>
            <a href={result.metadata.website} target="_blank"
              dangerouslySetInnerHTML={{
                __html: result.metadata.website,
              }}
            />
```